components/Common/AvatarGroup/index.tsx

```diff
@@ -0,0 +1,62 @@
+'use client';
+import * as Avatar from '@radix-ui/react-avatar';
+import { useState } from 'react';
+import type { FC } from 'react';
+import styles from './index.module.css';
+
+type AvatarGroupProps = {
+  avatars: {
```

Comment:
Make an individual type for Avatar.

Reply:
in separated file ?

Reply:
Nope

Reply:
OK 👍

Reply:
Still waiting you to make a dedicated type for this 👀